cosmos-cmd-tlm-api/app/controllers/narrative_controller.rb

```diff
@@ -79,7 +79,7 @@ def search
       start, stop = parse_time_input(x_start: params[:start], x_stop: params[:stop])
       q = params[:q]
       raise NarrativeInputError "Must include q value" if q.nil?
-      model_array = @model_class.get(scope: params[:scope], start: start, stop: stop)
+      model_array = @model_class.range(scope: params[:scope], start: start, stop: stop)
       model_array.find { |model| model.description.include? q }
```

Comment:
what is q?

Reply:
comment says "The string to contain in the narrative description". Worst variable name ever though.

Reply:
Fixed